cljfmt/src/cljfmt/main.clj

```diff
@@ -0,0 +1,189 @@
+(ns cljfmt.main
+  "Functionality to apply formatting to a given project"
+  (:require [cljfmt.core :as cljfmt]
+            [clojure.edn :as edn]
+            [clojure.java.io :as io]
+            [clojure.string :as str]
+            [clojure.stacktrace :as st]
+            [clojure.tools.cli :as cli]
+            [cljfmt.diff :as diff]
+            [meta-merge.core :refer [meta-merge]]))
+
+(def default-config
+  {:project-root "."
+   :file-pattern #"\.clj[csx]?$"
+   :paths []
+   :ansi? true
+   :indentation? true
+   :insert-missing-whitespace? true
+   :remove-surrounding-whitespace? true
+   :remove-trailing-whitespace? true
+   :remove-consecutive-blank-lines? true
+   :indents cljfmt/default-indents
+   :alias-map {}})
+
+(defn- abort
+  [& msg]
```

Comment:
Can you put the arguments on the same line as the function name when there's room to do so?